src/SFA.DAS.Apprenticeships.Api.Client/StandardApiClient.cs

```diff
@@ -131,5 +132,13 @@ public async Task<IEnumerable<Standard>> GetStandardsByIdAsync(List<string> ids,
 			    return await RequestAndDeserialiseAsync<IEnumerable<Standard>>(request);
 		    }
 	    }
+
+        public async Task<IEnumerable<StandardProvider>> GetProvidersByStandardAndLocation(int standardCode, string postCode, int page = 1)
```

Comment:
Could you please add "Async" to the name and create a synchronous method as well?

Reply:
@dashton82